diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -243,14 +245,14 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
                 """
                     |fun foo(list: List<Bar>?) {
                     |    list!!.filterNotNull()
-                    |        .map { it.baz() }.firstOrNull {
+                    |        .map { it.baz() }
```

Comment:
Why?

Reply:
I mean, there already was a warning for this line, why convert negative test to positive?